src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -25,8 +26,18 @@ public CommandResult execute() throws CommandException {
             throw new CommandException(MESSAGE_FAILURE);
         }
 
+        String commandString = undoRedoStack.peekRedo().toString();
+        String feedbackToUser = parseCommand(commandString);
         undoRedoStack.popRedo().redo();
-        return new CommandResult(MESSAGE_SUCCESS);
+        return new CommandResult(feedbackToUser);
+    }
+
+    /**
+     * Parses the output command to display the previously undone command
+     */
+    public static String parseCommand(String commandString) {
+        String output = String.format(MESSAGE_SUCCESS_FULL, commandString);
```

Comment:
SUCCESS_FULL?

Reply:
Not misspelled haha! What I meant was that this is the full form of MESSAGE_SUCESS. I shall rename it to FULL_MESSAGE_SUCCESS to avoid confusion!